### PR TITLE
Mark 2.22.1 aws-infrastructure release

### PR DESCRIPTION
**Pull request by kkulesza2**
Mark 2.22.1 release

### CODE REVIEW

**Review by mzyromski-atlassian — CHANGES_REQUESTED** [review]
This PR seems to also contain commits from already merged #105.

**Review by mzyromski-atlassian — COMMENTED** [review]
Sorry for nitpicking, but we usually prefer having a linear Git history, hence it would be better if you rebased your commit on `master` instead of merging.

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,9 +21,15 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.22.0...master
- - Fix regular expression for database url customization with `default_storage_engine`.
+[Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.22.1...master
 
+## [2.22.1]
```
Comment (mzyromski-atlassian) [inline CHANGELOG.md:26]: Date missing

Comment (kkulesza2) [reply CHANGELOG.md:26]:
```suggestion
## [2.22.1] 2020-11-27
```

**Comment by mzyromski-atlassian** [comment]
> Sorry for nitpicking, but we usually prefer having a linear Git history, hence it would be better if you rebased your commit on `master` instead of merging.

OK, let's assume "squash and merge" will do it's job here. 🤞

**Review by mzyromski-atlassian — APPROVED** [review]

**Comment by dagguh** [comment]
GitHub helps us enforce it:
image

**Pull request merged**